### PR TITLE
Update to USWDS

### PR DESCRIPTION
**Pull request by iamjolly**
First attempt at getting the Accessibility guide into the `uswds-jekyll` theme. 

We will want to check out all of the pages to ensure we preserve the functionality, content, and formatting from the prior theme. Also, is there a Federalist build of the branch folks can preview outside of their local dev environments?

### CODE REVIEW

File: _includes/components/search.html
Comment (iamjolly) [reply _includes/components/search.html:2]: hidden inputs don't need labels

File: _includes/components/search.html
Comment (iamjolly) [reply _includes/components/search.html:3]: hidden inputs don't need labels

File: _includes/components/search.html
Comment (tbaxter-18f) [reply _includes/components/search.html:2]: we should probably fix the accesslint bot on that point, huh?

**Comment by tbaxter-18f** [comment]
The preview on federalist: https://cg-06ab120d-836f-49a2-bc22-9dfb1585c3c6.app.cloud.gov/preview/18f/accessibility/update-to-uswds/

I think it all looks good, but I'd like to see other folks from the Accessibility Guild 👍

File: _includes/components/search.html
Comment (iamjolly) [reply _includes/components/search.html:2]: Yes, definitely. That's a false-positive, as `input`s that are hidden are not labelable per the HTML5 spec... https://www.w3.org/TR/html52/sec-forms.html#labelable-element

**Comment by iamjolly** [comment]
@Jacklynn @nkkl - Do you want to check out the preview of this on Federalist and let me know if anything looks broken or needs further refinement? Also, please add other reviewers, as needed, too. Thank you!

**Comment by Jacklynn** [comment]
For the most part things seem to be working but I think on pages with examples (https://accessibility.18f.gov/forms/) they're harder to see on the [new version](https://cg-06ab120d-836f-49a2-bc22-9dfb1585c3c6.app.cloud.gov/preview/18f/accessibility/update-to-uswds/forms/). So, 1) harder to see there's an example and 2) harder to see the example code. Would those be easy to update from engineering?

I also hesitate to overdo it, making a lot of changes, because the guild/others haven't reviewed this site in a while and we might want to integrate the content into [accessibility for teams](https://accessibility.digital.gov/). Thoughts @nkkl?

**Comment by nkkl** [comment]
I agree with Jacklynn's comments — the way code is presented in the old site, with syntax highlighting, is easier to follow. I also agree that it's probably not worth dumping a ton of effort into it. I will note that we're never sure exactly _when_ we'll get around to updating something, so we should probably make sure that we feel good about what we're publishing now, because it might be there for a while.

**Comment by iamjolly** [comment]
Let me see if I can get that better syntax highlighting working here first, then I can take a look at cleaning up the form examples. I think there may just be some missing CSS for the syntax highlighting, which shouldn't be too time-consuming.

Also, agree we should review this resource as a whole, make updates where needed, and incorporate the excellent Accessibility for Teams wherever we can. Thanks for these comments @Jacklynn and @nkkl.

**Comment by iamjolly** [comment]
@Jacklynn @nkkl - I've updated the styling for:
- code highlighting more like before
- fieldset/legend borders added
- fixed some margin issues with input/label combos
- improved formatting of radio buttons

Let me know if this looks good to merge, or if we should look at any problems on any other pages before doing that. 

Screen Shot 2019-11-25 at 10 40 33 AM

**Comment by iamjolly** [comment]
Updated Federalist build of this branch is at: https://cg-06ab120d-836f-49a2-bc22-9dfb1585c3c6.app.cloud.gov/preview/18f/accessibility/update-to-uswds/forms/

**Comment by Jacklynn** [comment]
Amazing! thanks so much for all this work @iamjolly ✨

**Review by Jacklynn — APPROVED** [review]
This looks great, much easier to use than the default styles that ported over!

**Pull request merged**
